### DISCUSSION TITLE
Transcribe only certain portions of source media (esp Youtube)

### DISCUSSION THREAD

**Discussion by suckerfish** [ggml-org/whisper.cpp#2028] [Ideas]
I don't think this currently exists but the videos I watch often only require transcription of certain portions of the video. For example I like watching movie/TV reactions and they often have their thoughts at the end. I just want to transcribe the last 15-20 minutes of a video, in some cases. 

it would be nice if I could enable the ability to specify the start and end timestamps for what I want to transcribe. And it would be great if it could be per video in the batch mode, too.

**Comment by ggerganov** [comment]
Already possible - see the `--offset-t` and `--duration` CLI args

**Reply by suckerfish** [reply]
My apologies. My intent was to make the suggestion specifically for the Macwhisper app to have this functionality so I may be in the wrong place for this kind of suggestion.

**Comment by Fran2789** [comment]
you might want to download youtube's transcriptions
